retriever/lib/engine_tools.py

```diff
@@ -112,21 +112,30 @@ def reset_retriever(scope="all", ask_permission=True):
     }
 
     path = os.path.normpath(HOME_DIR)
-    warn_msg = warning_messages[scope].format(path)
-    if ask_permission:
-        confirm = input(warn_msg)
-        while not (confirm.lower() in ['y', 'n', '']):
-            print("Please enter either y or n.")
-            confirm = input()
+    if scope in ['all', 'scripts', 'data']:
```

Comment:
Thanks @harshitbansal05, let us reduce the hard coding here ;
We can either create the path values at  144 
```python
    rw_dir = os.path.normpath(os.path.join(HOME_DIR, 'raw_data'))
    sc_dir = os.path.normpath(os.path.join(HOME_DIR, 'scripts'))
    script_py_form = os.path.normpath(os.path.join(sc_dir, "{script}.py"))
    script_json_form  = os.path.normpath(os.path.join(sc_dir, "{script}.json"))
    # Or after... script = scope.replace('-', '_')
    script_path_py = os.path.normpath(os.path.join(sc_dir, script + ".py"))
    script_path_json = os.path.normpath(os.path.join(sc_dir, script + ".json"))
```
 
or decide to 
import 
`from retriever.lib.defaults import HOME_DIR, SCRIPT_WRITE_PATH, DATA_WRITE_PATH`
In this case the rw_dir is same as DATA_WRITE_PATH and sc_dir = SCRIPT_WRITE_PATH.
If you decide to import make sure that there are no cyclic imports caused.
I think both ways are good enough